src/WikiService.Business/Commands/Article/Edit/EditArticleRequestValidator.cs

```diff
@@ -96,7 +103,20 @@ private async Task<bool> DoesRubricIsActiveAsync(Guid rubricId)
               : false,
             "This rubric doesn't exist."
           }
-        });
+        });
+      #endregion
+
+      #region Position
+      AddFailureForPropertyIf(
+       nameof(EditRubricRequest.Position),
+       x => x == OperationType.Replace,
+       new()
+       {
+         {
+           x => int.Parse(x.value?.ToString()) > 0,
+           "Position must be greater than 0."
```

Comment:
why?

Reply:
The position is meant by a natural number.